### PR TITLE
Save caps

### PR DESCRIPTION
**Pull request by noodlemoodle**
nano pr https://github.com/nanoPayinc/NANOPAY/pull/9493

### CODE REVIEW

File: src/foam/nanos/crunch/ui/ScrollSectionWizardView.js [hunk 1]
```diff
@@ -83,9 +97,26 @@ foam.CLASS({
     },
     {
       name: 'save',
-      code: function(x) {
-        console.log('saving');
-        this.stack.back();
+      code: async function(x) {
```
Comment (Nauna) [inline src/foam/nanos/crunch/ui/ScrollSectionWizardView.js]: personally don't like the style of making a function 'async' in foam. It changes the functional return to a promise, and I don't see it being necessary here.

in your 'capabilityDAO.then' you can set your success notify and stack back ... and then do a 'catch' after with a notify with an error msg:
 Example. ```
.catch((e) => this.notify(this.ERROR_MSG + `${e && e.message ? ': ' + e.message : (e ? ': ' + e : '')}`, 'error'));
```

File: src/foam/nanos/crunch/ui/ScrollSectionWizardView.js [hunk 2]
```diff
@@ -83,9 +97,26 @@ foam.CLASS({
...
+
+        var capabilityDAO = x.capabilityDAO;
```
Comment (Nauna) [inline src/foam/nanos/crunch/ui/ScrollSectionWizardView.js]: there shouldn't be a reason to import capabilityDAO if you just pull it from the context here. Same for the userCapabilityJunctionDAO.

File: src/foam/nanos/crunch/ui/ScrollSectionWizardView.js [hunk 3]
```diff
@@ -83,9 +97,26 @@ foam.CLASS({
...
+        var userCapabilityJunctionDAO = x.userCapabilityJunctionDAO;
+
+        this.sectionsList.forEach((m) => {
+          capabilityDAO
```
Comment (Nauna) [inline src/foam/nanos/crunch/ui/ScrollSectionWizardView.js]: we are going through each capability in src/foam/u2/crunch/CrunchController.js and think its somewhat redundant to go through it again here.

I'm needing to collect more data from the capabilities as well and have just been modifying the sectionsList.
take a look at https://github.com/Nauna/foam2/pull/13/files#diff-254bba599b3165e8c6c557e8b52a0c19R101

tell me what you think. Look at the argsList, ofList, and the daoList that is getting set.

ps. we need to name sectionsList to something better - suggestions??

File: src/foam/nanos/crunch/ui/ScrollSectionWizardView.js (hunk 2 above)
Comment (noodlemoodle) [reply src/foam/nanos/crunch/ui/ScrollSectionWizardView.js]: true, removed from imports and will use this

**Pull request merged**